src/base/tests/test_util.py

```diff
@@ -700,22 +700,22 @@ def test_explode_mult_filters(self):
     def test_explode_query_range(self):
         cr = self.env.cr
 
-        cr.execute("SELECT count(id) FROM res_partner_title")
+        cr.execute("SELECT count(id) FROM res_partner")
```

Comment:
Cannot use res_partner here. It fails in 12.3
```
ERROR: test_explode_query_range (odoo.addons.base.maintenance.migrations.base.tests.test_util.TestPG)
...
psycopg2.IntegrityError: update or delete on table "res_partner" violates foreign key constraint "res_partner_commercial_partner_id_fkey" on table "res_partner" 
```

Maybe keep using `res_partner_bank`?

Reply:
Used res_partner_category to avoid creating a partner for the bank